packages/apputils/src/semanticCommand.ts

```diff
@@ -37,6 +37,11 @@ export class SemanticCommand {
    */
   static readonly DEFAULT_RANK = 500;
 
+  /**
+   * The `args` key for a semantic command's current widget ID.
+   */
+  static readonly WIDGET = 'semantic-widget';
```

Comment:
This is a nitpick, but for "special" args it looks like we usually follow camelCase:
- `_luminoEvent` (https://github.com/jupyterlab/lumino/pull/644)
- `isPalette` https://github.com/jupyterlab/jupyterlab/pull/17258

```suggestion
  static readonly WIDGET = 'semanticWidget';
```